lib/ui/colorize.js

```diff
@@ -26,7 +26,7 @@ var styles = {
     'bgDefault': ['\033[49m', '\033[49m']
 };
 
-var enabled = !process.env.NOCOLOR && tty.isatty(1) && tty.isatty(2);
+var enabled = process.env.COLOR || (!process.env.NOCOLOR && tty.isatty(1) && tty.isatty(2));
```

Comment:
Seems in this line checks about file descriptors point to TTY are being ignored if `process.env.COLOR` is being set. This may lead to case when color output may occur, for example, in log file. Looks like this is not a correct behavior, output anywhere expect TTY has to be clean, without colors.